BLL/ReviewService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace BLL
+{
+    public class ReviewService
+    {
+        public bool EditReview(Drink drink, int score, string review)
```

Comment:
I don't think we have to create new class for this method because we could include it in DrinkReview.cs